File: banking/custom/employee.py
import frappe
from frappe import _


def validate(doc, event):
	if hasattr(doc, "iban") and doc.has_value_changed("iban") and doc.iban:
		if frappe.db.exists(
			"Bank Account",
			{"party_type": "Employee", "party": doc.name, "disabled": 0},
		):
			frappe.msgprint(
				_(
					"The IBAN in this form will not be used in SEPA Payment Orders because a Bank Account is already linked to this Employee."
				),
				title=_("IBAN Warning: Not used in SEPA Payment Orders"),
				indicator="orange",
			)

File: banking/custom/employee.js
frappe.ui.form.on("Employee", {
	setup(frm) {
		frm.make_methods = frm.make_methods || {};
		frm.make_methods["Bank Account"] = () => frm.trigger("make_bank_account");
	},
	make_bank_account(frm) {
		banking.utils
			.create_party_bank_account("Employee", frm.doc.name)
			.then(() => {
				frappe.show_alert({
					message: __("Employee Bank Account was created."),
					indicator: "green",
				});
			})
			.catch(() => {
				frappe.show_alert({
					message: __("Employee Bank Account was not created."),
					indicator: "yellow",
				});
			});
	},
});
